api/src/modules/sourcing-record-groups/sourcing-record-groups.entity.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
File duplicated? There are 2 `sourcing-record-groupsS` entities in this PR

Reply:
*facepalm*